src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/main.js

```diff
@@ -580,6 +591,7 @@ define(['services/husky/util'], function(util) {
                     this.sandbox.dom.append(ul, _.template(templates.contentItem)({
                         dataId: item[this.options.idKey],
                         value: item[this.options.titleKey],
+                        image: item[this.options.imageKey] ? item[this.options.imageKey] : null,
```

Comment:
`item[this.options.imageKey] || null` would be cleaner, wouldn't it? And doesn't the value already contain null anyway?